booq-dev/deployment.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Deployment
+metadata:
+  name: booq
+  labels:
+    app: booq
+
+spec:
+  selector:
+    matchLabels:
+      app: booq
+  template:
+    metadata:
+      labels:
+        app: booq
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: e505.tokyotech.org
+
+      containers:
+
+        - name: booq-frontend
+          image: ghcr.io/traptitech/booq-ui:latest
+          ports:
+            - containerPort: 80
+
+        - name: booq-backend
+          image: ghcr.io/traptitech/booq:latest
+          env:
+            - name: HOST
+              value: "booq.trap.jp"
+            - name: MYSQL_HOST
+              value: "private.kmbk.tokyotech.org"
+            - name: OS_AUTH_URL
+              value: "https://identity.tyo1.conoha.io/v2.0"
+          envFrom:
+            - secretRef:
+                name: booq-secrets
```

secretの名前を、ここを変更するかSecret側を変更するかで合わせて欲しいです🙏
```suggestion
                name: booq-secret
```